FlowCrypt/Controllers/Setup/SetupInitialViewController.swift

```diff
@@ -123,7 +120,8 @@ extension SetupInitialViewController {
 
         Task {
             do {
-                let keys = try await service.fetchBackupsFromInbox(for: user)
+                let userId = UserId(email: appContext.user.email, name: appContext.user.name)
```

Comment:
This initializer is used in many places, it'll be better to replace it with computed property on `AppContextWithUser`:
```swift
var userId: UserId {
  UserId(email: user.email, name: user.name)
}
```

Then we'll be able to use it with just `appContext.userId`

Reply:
Since this object is constant I decided to go with usual property.